### PR TITLE
Update README

### PR DESCRIPTION
**Pull request by apiraino**
Just a little more detail that helped me (as a total newbie) to clear up things when trying to complile the examples.

make sense this stuff?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,13 +6,25 @@ A few gtk-rs examples. To build, just do:
...
 ```
 
-or to enable GTK 3.10 examples as well:
+or to enable GTK 3.1x, 3.2x, all examples as well:
 
 ```Shell
-> cargo build --features gtk_3_10
+> cargo build --features gtk_3_18
+> cargo build --features gtk_3_22_30
+> cargo build --all-features
```
Comment (EPashkin) [inline README.md]: IMHO one version like 3.18 is enough.

Comment (GuillaumeGomez) [reply README.md]: I agree with @EPashkin in here.

File: README.md [hunk 2]
```diff
@@ -6,13 +6,25 @@ A few gtk-rs examples. To build, just do:
 > cargo build
 ```
 
-or to enable GTK 3.10 examples as well:
+or to enable GTK 3.1x, 3.2x, all examples as well:
```
Comment (GuillaumeGomez) [inline README.md]: What about "or to enable GTK 3.x depending on the version needed by the example:"

File: README.md [hunk 3]
```diff
@@ -6,13 +6,25 @@ A few gtk-rs examples. To build, just do:
...
+> cargo build --all-features
 ```
 
-And then run the executables. Please be sure to have installed all the required libraries before building examples (the list is available on [gtk](https://github.com/gtk-rs/gtk/).
+And then run the executables with:
+
+``` Shell
+./target/debug/EXAMPLE-NAME
+```
+or with cargo run (repeating the compilation features used above), example:
```
Comment (GuillaumeGomez) [inline README.md]: Please add an empty line before and after this one.

File: README.md [hunk 4]
```diff
@@ -6,13 +6,25 @@ A few gtk-rs examples. To build, just do:
...
+``` Shell
+cargo run --all-features --bin EXAMPLE-NAME
+```
+
+Please be sure to have installed all the required libraries before building examples (the list is available on [gtk](https://github.com/gtk-rs/gtk/).
```
Comment (GuillaumeGomez) [inline README.md]: There is a missing closing paren at the end. :)

Comment (GuillaumeGomez) [reply README.md]: Also, I'd replace "\[gtk\]" with "\[gtk-rs\]" to make sure people don't confuse it with GTK official repository.

**Comment by GuillaumeGomez** [comment]
Makes sense, indeed! Just a few nits and we're good to go.

File: README.md (hunk 1 above)
Comment (apiraino) [reply README.md]: Ok, I'd like to reword that so the reader understands which are the options available and how to find it. E.g. "check Cargo.toml [features] to see all specific GTK compile features available"

Comment (GuillaumeGomez) [reply README.md]: To "enable them all", you can just enable the highest one. Also, in the cargo file, there is not only libraries versions handling but also features enablement. So I really like the first part of the sentence but way less the second one. ;)

**Comment by apiraino** [comment]
done here, too (hoping to having correctly interpreted your suggestion)

**Comment by GuillaumeGomez** [comment]
Thanks!

**Pull request merged**